Dockerfile

```diff
@@ -5,6 +5,7 @@ RUN make build
 
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
 COPY --from=builder /go/src/github.com/openshift/cluster-samples-operator/cluster-samples-operator /usr/bin/
+RUN ln -sf /usr/bin/cluster-samples-operator /usr/bin/cluster-samples-operator-watch
```

Comment:
Use hard link - I learned this the hard way with @ricardomaraschini.

Reply:
updating momentarily